tests/pytorch/mock_sparse/test_elementwise_op_sp.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import pytest
+import dgl
+import dgl.backend as F
+import torch
+import numpy
+import operator
+from dgl.mock_sparse import elementwise_op_sp
+from dgl.mock_sparse.sp_matrix import SparseMatrix
```

Comment:
Users shouldn't import from `dgl.mock_sparse.elementwise_op_sp` or `dgl.mock_sparse.sp_matrix`. They should directly import `dgl.mock_sparse.some_api` or `dgl.mock_sparse.SparseMatrix`. This is an issue related to the `__all__` comment.